### DISCUSSION TITLE
About inp mesh import problem

### DISCUSSION THREAD

**Discussion by Gallasd** [idaholab/moose#21493] [Q&A General] [answered]
Hello,
I don't know if I haven't understood how to import meshes in inp format into moose.
Initially I was only working on one geometry model and moose didn't recognize the node set I defined.By consulting other people's examples, I found that I can add this code to identify node sets
' construct_side_list_from_node_list=true'

In the follow-up, I need to consider multiple models, and moose reports an error, saying that importing multiple parts is not yet supported.I tried to define the element set, but my element set is not recognized in the block.
I don't know how to make moose recognize my block, can you help me, thank you very much!

The reason why I consider defining the unit set to represent the block is because when I build a single model study, the elemtent set is defined, and the block can be seen in the paraview.
![image](https://user-images.githubusercontent.com/107783970/177162187-d5f87f32-78a9-4d2c-8598-7c30f10c6f7a.png)
![image](https://user-images.githubusercontent.com/107783970/177162701-acb45241-709d-4ec4-9532-51abe6e166f2.png)
But it is not recognized in moose

**Comment by GiudGiud** [comment, marked as answer]
Hello

The Abaqus file mesh reader is actually inherited from libmesh.
You may look at the code there to see what it does
https://libmesh.github.io/doxygen/classlibMesh_1_1AbaqusIO.html#a4935543726ef7a56278f486d2904f423
otherwise you may want to ask in the libmesh discussions

Guillaume

**Reply by Gallasd** [reply]
I want to ask you, does the block in moose correspond to the unit set in abaqus? If this is uncertain, I can't find the direction I should go to solve

**Reply by GiudGiud** [reply]
looking at the code it seems to correspond to "elset"
I dont see any reference to unit sets there
